### ISSUE TITLE
GUI is unreadable when using a dark High Contrast theme

### ISSUE THREAD

**Issue by RedMser** [lucasg/Dependencies#222]
![image](https://user-images.githubusercontent.com/5117197/176449027-eea0e1ef-fa78-4954-8ad5-47260903ddd0.png)

All of the ListViews seem to hard-code the white background color, while the font color gets overridden by the dark high contrast theme to be white as well. This makes the text completely unreadable, unless rows are selected.

Ideally, colors should be configurable. But as an alternative, ensure that the background color uses a system color, while the text keeps current behavior of following the theme.

**Comment by oscarbg** [comment]
how you change the theme?

**Comment by RedMser** [comment]
> how you change the theme?

This is talking about the operating system's theme, and not a theme option built into the software.

**Comment by oscarbg** [comment]
yep.. but I'm in dark mode and I see doesn't apply any dark theme..

**Comment by RedMser** [comment]
> yep.. but I'm in dark mode and I see doesn't apply any dark theme..

Dark theme I don't know, but I'm talking about a High Contrast dark theme, which is used for accessibility and renders differently from Windows' regular dark theme.

**Comment by oscarbg** [comment]
can share how to change to this High Contrast dark theme in Windows?

**Comment by RedMser** [comment]
> can share how to change to this High Contrast dark theme in Windows?

https://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025#ID0EBD=Windows_10

**Comment by hfrmobile** [comment]
To summarize:

- Light Theme --> No problem
- Dark Theme --> No problem
- High Contrast theme --> Problem as shown in the screenshot from original poster

I was searching for "dark mode" to ensure that there not already an item about it to avoid duplicates. It seems that Dependencies does not support any theming and this thread is really a bug, not a feature request.
